bench/wtperf/wtperf.c

```diff
@@ -109,6 +112,18 @@ generate_key(CONFIG *cfg, char *key_buf, uint64_t keyno)
 	sprintf(key_buf, "%0*" PRIu64, cfg->key_sz - 1, keyno);
 }
 
+static inline uint64_t
+decode_key(CONFIG *cfg, char *key_buf)
+{
+	char *endPtr;
+	uint32_t i;
+	i= 0;
+	for (;i < cfg->key_sz; i++)
```

Comment:
Is this loop necessary? Does strtoull already deal with leading `0` characters? Do we get key inputs with leading ```NULL``` characters?